internal/core/command/controller/messaging/router.go

```diff
@@ -0,0 +1,35 @@
+//
+// Copyright (C) 2022 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package messaging
+
+// MessagingRouter defines interface for Command Service to know
+// where to route the receiving device command response.
+type MessagingRouter interface {
+	// Get returns the responseTopicPrefix by RequestId
+	Get(string) string
+	// Set sets the responseTopicPrefix with RequestId as the key
+	Set(string, string)
```

Comment:
Get and Set names are too genetic give they are on a router. Getters functions in Go are just the attribute name. Strange, but... ;-) 
```suggestion
        // ResponseTopicPrefix returns the responseTopicPrefix by RequestId
	ResponseTopicPrefix(requestId) string
	// SetResponseTopicPrefix sets the responseTopicPrefix with RequestId as the key
	SetResponseTopicPrefix(requestId string, topic string)
```